packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -103,6 +110,27 @@ class Statement extends React.Component {
 
         if (error) return <p>{error}</p>;
 
+        const account_statistics_component = (
+            <React.Fragment>
+                <div className='statement__accountStatistics'>
+                    <div className='statement__accountStatistics--Rectangle'>
```

Comment:
(throughout)
```suggestion
                    <div className='statement__account-statistics--is-rectangle'>
```

- Class names should be in kebab-case format. 
- Modifiers usually are boolean, if this is a rectangle, let's use `is-rectangle` modifier.